packages/sit-onyx/src/components/OnyxDataGrid/OnyxDataGrid.stories.ts

```diff
@@ -0,0 +1,21 @@
+import type { Meta, StoryObj } from "@storybook/vue3";
+import OnyxDataGrid from "./OnyxDataGrid.vue";
+
+const meta: Meta<typeof OnyxDataGrid> = {
+  title: "DataGrid/DataGrid",
```

Comment:
```suggestion
  title: "Data/DataGrid",
```

Wouldn't the data category be more fitting here? Would also be aligned with Figma then